### ISSUE TITLE
Add more basemaps to adhoc map search and location filters

### ISSUE THREAD

**Issue by MortenHofft** [gbif/portal16#1488]
Ad hoc search would benefit from more base maps. Since we use WGS84 to include the poles, our choices are somewhat limited though.

**Comment by MattBlissett** [comment]
The GBIF Natural or OSM Bright layers: https://tile.gbif.org/ui/4326/ (on the dropdown under EPSG:4326 Raster) have much more detail than the almost-empty maps used at the moment for the filter map or the results map.

**Comment by MortenHofft** [comment]
Yes, I imagined adding those we have (above included) for now. They each have their merit.

**Comment by MattBlissett** [comment]
It would also be useful to use the language-specific version of the map on ad-hoc search. Our other pages with maps do this.

(And the map style comparison, while I'm here: https://tile.gbif-uat.org/3857.html#map3857=8/41445124.17/7484625.85/0 )
